ajax/libs/xstream/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "xstream",
+  "version": "7.0.0",
+  "description": "An extremely intuitive, small, and fast functional reactive stream library for JavaScript",
+  "typings": "index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/staltz/xstream.git"
+  },
+  "author": "Andre Staltz <andre+npm@staltz.com> (http://andre.staltz.com/)",
+  "license": "MIT",
+  "keywords": [
+    "xstream",
+    "reactive stream",
+    "functional"
+  ],
+  "dependencies": {
+    "symbol-observable": "^1.0.2"
+  },
+  "config": {
```

Comment:
Sorry that we can not recognize all the useless fields right now, may need to use something like whitelist in future, sorry for the inconvenience caused.